dvc/command/gc.py

```diff
@@ -46,7 +46,10 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    GC_HELP = "Collect garbage."
+    GC_HELP = (
+        "Collect garbage from your DVC cache. \n"
+        "Deletes all files in the cache which are not in use by the specified git references (defaults to just HEAD)."
```

Comment:
minor change - could you make it fit into 80 symbols, please? Or even better install the pre-commit black hook and run it through it like @efiop mentioned.

Reply:
@shcheklein This is after the black hook (it added the parens around the string).
I'll see if I can find shorten it.

Reply:
Right, we accidentally ignored this in our flake8 config and black doesn't seem to care about line length limit for strings. Looking into it.

Reply:
Ok, fixed our flake8 defaults. @guysmoilov Could you please rebase? I.e.
```
git remote add upstream https://github.com/iterative/dvc
git fetch upstream
git rebase upstream/master
git push -f
```

Reply:
@efiop Done